packages/clay-empty-state/src/EmptyState.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Message the user will see describing what they can do when on this screen
+	 */
+	description?: string;
+
+	/**
+	 * Source of the image to signify the state
+	 */
+	imageSrc?: string;
+
+	/**
+	 * Title of the message highlighting the description
+	 */
+	title?: string;
+}
+
+const ClayEmptyState: React.FunctionComponent<IProps> = ({
+	children,
+	description = 'Sorry, there are no results found',
+	imageSrc,
+	title = 'No results found',
+}) => (
+	<div
+		className={`empty-state-root ${!imageSrc &&
```

Comment:
You can use our `classNames` dependency we use in a lot of other packages and convert this to.

```jsx
className={classNames('empty-state-root', {
	'empty-state-root--no-image': !imageSrc,
})}
```